cases/withdraw.test.js

```diff
@@ -30,15 +32,19 @@ describe("Withdraw", () => {
       },
     });
     infoJSON = await infoResponse.json();
-    const currencies = Object.keys(infoJSON.withdraw);
-    enabledCurrency = currencies.find(
-      (currency) => infoJSON.withdraw[currency].enabled,
-    );
+    currencies = Object.keys(infoJSON.withdraw);
+    enabledCurrency = testCurrency
+      ? testCurrency
+      : currencies.find((currency) => infoJSON.withdraw[currency].enabled);
```

Comment:
Can we pull this all out into a helper instead of copying it to each file?

Reply:
Yup, sure

Reply:
Done, @msfeldstein :)